exporters/otlp/otlptrace/otlptracegrpc/mock_collector_test.go

```diff
@@ -0,0 +1,239 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Copied and removed metrics part from: exporters/otlp/otlpgrpc/mock_collector_test.go